libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtpPacket.java

```diff
@@ -246,6 +302,9 @@ public static RtpPacket parse(ParsableByteArray packetBuffer) {
         .setTimestamp(timestamp)
         .setSsrc(ssrc)
         .setCsrc(csrc)
+        .setExtension(hasExtension)
```

Comment:
I don't think its necessary to save the header extension data. None of the RTSP readers are utilizing this data at the moment and its just adding cruft to the class. I think its enough to just parse the extension if it exists as to not have the readers choke on the payload.

Reply:
@microkatz 

True, good suggestions.
So I can also remove saving "`headerExtension`" (thats's also misleading) and `extensionPayload` data? Which means I can also delete second Unit test I added `buildRtpPacketWithHeaderExtension_matchesPacketData`. And If I don't save hasExtension I will not check it in first Unit test I added `parseRtpPacketWithHeaderExtension` ?

Reply:
The reason why I mentioned the RtpH264ReaderTest was to have a unit test parsing an RTP packet with both a header extension and a payload. In this case, without your changes, the Rtp parsing logic would incorrectly group the headerextension into the general payload. If you can create a unit test in RtpPacketTest that exemplifies this then that is fine as well.